docs/models/operations/CreateBlocklistIdentifierRequestBody.md

```diff
@@ -3,6 +3,6 @@
 
 ## Fields
 
-| Field                                                                                                         | Type                                                                                                          | Required                                                                                                      | Description                                                                                                   |
-| ------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------- |
-| `identifier`                                                                                                  | *String*                                                                                                      | :heavy_check_mark:                                                                                            | The identifier to be added in the block-list.<br/>This can be an email address, a phone number, or a web3 wallet. |
\ No newline at end of file
+| Field           | Type        | Required                | Description                                                                                                       |
+| --------------- | ----------- | ----------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `identifier`    | *String*    | :heavy_check_mark:      | The identifier to be added in the block-list.<br/>This can be an email address, a phone number, or a web3 wallet. |
```

Comment:
I'm not sure what's enforcing such incredibly wide tables, it seems really excessive and makes them incredibly painful to read in any editor...